code/config.ss

```diff
@@ -103,8 +103,8 @@
      ((slug . phone-number)
       (uuid . "8ed74044-2361-459e-aca5-abd73ed5c1cb")
       (core . #f)
-      (unlocked-by)
-      (difficulty . 1)
+      (unlocked-by raindrops)
```

Comment:
This has to be unlocked by a core exercise.

Also, the unlocked by syntax has to be:
``` scheme
(unlocked-by . dependency)
```
Otherwise it will render a json array instead of a key value pair